features/other_b/subscriptions.feature

```diff
@@ -259,3 +259,26 @@
     And I go to my subscriptions page
     And I press "Unsubscribe from I am <strong>er Than Yesterday & Other Lies"
   Then I should see "You have successfully unsubscribed from I am <strong>er Than Yesterday & Other Lies"
+
+  Scenario: Subscription emails include HTML in work and chapter summaries in the HTML email and convert it for plain text emails.
```

Comment:
I'd prefer this in user_mailer_spec.rb. We don't need to add new cuke steps in that case.

Reply:
Unfortunately, I did it this way because I could not figure out how to get subscription emails working in specs. /o\